qt/manageprofiles/tab_options.py

```diff
@@ -83,6 +85,23 @@ def __init__(self, parent):
             _('Create a new backup whether there were changes or not.'))
         tab_layout.addWidget(self.cbTakeSnapshotRegardlessOfChanges)
 
+        # warn free space
+        hlayout = QHBoxLayout()
+        tab_layout.addLayout(hlayout)
+
+        WARN_FREE_SPACE_UNITS = {
+            config.Config.DISK_UNIT_MB: 'MiB',
+            config.Config.DISK_UNIT_GB: 'GiB'
+        }
+        self.suWarnFreeSpace = SpinBoxWithUnit(
+            self, (1, 99999), WARN_FREE_SPACE_UNITS)
+
+        self.cbWarnFreeSpace = StateBindCheckBox(
+            _('Show warning if the free space is less than'), self)
```

Comment:
```suggestion
            _('Warn if the free disk space falls below'), self)
```

I am not very good in English. "...is less than" would also be OK for me.

Reply:
I've made the suggested change.